### PR TITLE
[Path] Millface & drilling improvements

### PR DESCRIPTION
**Pull request by Schildkroet**
Added 'Clear Edges' to GUI in MillFace OP
millface

Removed 'Use Tip Length' in Drilling OP and added 'Extra Offset' instead.
drill

### CODE REVIEW

**Comment by sliptonic** [comment]
Looks fine to me.

File: src/Mod/Path/PathScripts/PathDrilling.py [hunk 1]
```diff
@@ -218,6 +222,7 @@ def SetupProperties():
     setup.append("DwellEnabled")
     setup.append("AddTipLength")
     setup.append("ReturnLevel")
+    setup.append("ExtraDepth")
```
Comment (Russ4262) [inline src/Mod/Path/PathScripts/PathDrilling.py]: Should this be `setup.append("ExtraOffset")` ?

File: src/Mod/Path/PathScripts/PathDrilling.py [hunk 2]
```diff
@@ -75,6 +75,8 @@ def initCircularHoleOperation(self, obj):
         obj.addProperty("App::PropertyEnumeration", "ReturnLevel", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "Controls how tool retracts Default=G99"))
         obj.ReturnLevel = ['G99', 'G98']  # Canned Cycle Return Level
         obj.addProperty("App::PropertyDistance", "RetractHeight", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "The height where feed starts and height during retract tool when path is finished while in a peck operation"))
+        obj.addProperty("App::PropertyEnumeration", "ExtraOffset", "Drill", QtCore.QT_TRANSLATE_NOOP("App::Property", "How far the drill depth is extended"))
+        obj.ExtraOffset = ['None', 'Tip Length', '2x Tip Length']  # Canned Cycle Return Level
```
Comment (Russ4262) [inline src/Mod/Path/PathScripts/PathDrilling.py]: Please set the default value for "ExtraOffset" in the `opSetDefaultValues()` method later in the module.

**Comment by Schildkroet** [comment]
I'm thinking of changing "Tip Length" to "Drill Tip" since it suits the new option somehow better. What do you think?

**Comment by Russ4262** [comment]
> I'm thinking of changing "Tip Length" to "Drill Tip" since it suits the new option somehow better. What do you think?

I think the latter, 'Drill Tip' would be fine.  However, I suggest a change in the property and display name from 'ExtraOffset' to the previous 'ExtraDepth' or a similar 'ExtendDepth.'  When I think of the word, offset, I generally think lateral motion, not vertical.  Whereas, using one of the two suggested names will clarify the direction because of the inclusion of the word, depth.  Nonetheless, I defer to those who have drilling experience.  I have only used a hand drill in my life, never a computer-controlled drilling machine. I am really not familiar with much commercial or industrial standard terminology for most of the PathWB's operations.

Glad to have you coding for the PathWB.

**Comment by Schildkroet** [comment]
Changed names to "Extend Depth" and "Drill Tip" since i also was not satisfied with the old names

**Pull request merged**